trino/sqlalchemy/util.py

```diff
@@ -0,0 +1,90 @@
+import json
+from urllib.parse import quote_plus
+
+from typing import Optional, Dict, List, Union, Tuple
+from sqlalchemy import exc
+from sqlalchemy.engine.url import _rfc_1738_quote  # noqa
+
+
+def _url(
```

Comment:
Some parameters are missing like `verify` or `redirect_handler`

Reply:
OAuth is not yet supported in the url. I think we can leave that for another PR. I have added verify

Reply:
Sure, just please create a follow-up issue for it after merging, Kerberos doesn't seem supported as well, right?

Reply:
Correct I supported only the fields that are currently parsed in the sqlalchemy connection string (and added `verify`).